punx/tests/test_issue129.py

```diff
@@ -0,0 +1,106 @@
+import h5py
+import numpy as np
+import os
+
+from punx.tests._core import hfile
+from punx import h5tree
+
+
+def test_render_multiple_axes_attribute(hfile):
+    """Ensure axes attributes are rendered as list of double quoted strings.
+
+    @axes should be saved as an array of byte strings or an array of object
+    strings because the NeXus standard says so. Single strings separated by
+    whitespace or other charachters will not be rendered correctly.
+    """
+    assert os.path.exists(hfile)
+    with h5py.File(hfile, "w") as h5:
+        nxentry = h5.create_group("Scan")
+        nxentry.attrs["NX_class"] = "NXentry"
+
+        nxdata = nxentry.create_group("data")
+        nxdata.attrs["NX_class"] = "NXdata"
+        nxdata.attrs["axes"] = "two_theta"
```

Comment:
Isn't **this** (`axes`) the attribute to be tested?

Reply:
You want me to only compare that one line in the report to the reference instead of all lines in the report?